REFERENCE.md

```diff
@@ -0,0 +1,242 @@
+# Reference
+
+## Resource types
+* [`zfs`](#zfs): Manage zfs. Create destroy and set properties on zfs instances.  **Autorequires:** If Puppet is managing the zpool at the root of this zfs in
+* [`zpool`](#zpool): Manage zpools. Create and delete zpools. The provider WILL NOT SYNC, only report differences.  Supports vdevs with mirrors, raidz, logs and s
+## Resource types
```

Comment:
Bug in Puppet Strings? I don't think we should have two H2s with the same name.

Reply:
it's either a bug in puppet strings or I've messed something up in how I'm documenting this module... I'm not quite sure which yet